help/develop/domains/chem/cheminformatics.md

```diff
@@ -97,10 +74,10 @@ similarity.
 Produces a Datagrok [`DataFrame`](https://datagrok.ai/js-api/classes/dg.DataFrame) of three
 columns ([code sample](https://public.datagrok.ai/js/samples/domains/chem/similarity-scoring-sorted)):
 
-* The 1-st column, named `molecule`, contains the original molecules string representation from the input `column`
-* The 2-nd column, named `score`, contains the corresponding similarity scores of the range from 0.0 to 1.0, and the
-  DataFrame is sorted descending by this column * The 3-rd column, named `index`, contains indices of the molecules in
-  the original input `column`
+* The 1-st column, named `molecule`, contains the original molecules from the input `column`
```

Comment:
Maybe we should remove `The 1-st/2-nd etc. column, named`?
I suggest:
`Molecule column conatins`
`Score column contains`
And lets put dots at the end of the sentence :)

Reply:
Done

Reply:
Done